course6/toyapp_thomasdhuys/drawlib/drawer.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <vector>
+#include "primitives.h"
+#include "render_object.h"
+
+namespace Draw {
+
+class Drawer {
+public:
+	static RenderObject Draw();
```

Comment:
The function doesn't really "draw" anything, it just returns a collection of vertices.